assignment1.txt

```diff
@@ -0,0 +1,17 @@
+1. A compiler is a program that takes code written in C and translates it into Assembly, a compiler is used so a computer can be one step closer to reading code
```

Comment:
right, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.